tests/test_return_types_legacy.py

```diff
@@ -22,6 +22,17 @@
 from pennylane.exceptions import QuantumFunctionError
 from pennylane.measurements import MeasurementProcess
 
+
+def _get_all_shots(shot_vector):
```

Comment:
Not too important but could we rename this `_get_num_shot_copies`?

Reply:
https://github.com/PennyLaneAI/pennylane/pull/7979/commits/37de1e13d32a65e09dd62993f2e1a41e48caeae9